src/api/spec/models/workflow_spec.rb

```diff
@@ -1,20 +1,18 @@
 require 'rails_helper'
 
-RSpec.describe Workflow, type: :model, vcr: true do
```

Comment:
There are no cassettes for the specs of this model. This can be safely removed since the backend isn't involved.